lib/SignIn.dart

```diff
@@ -0,0 +1,266 @@
+
+  @override
+  _MySignInPageState createState() => _MySignInPageState();
+}
+
+class _MySignInPageState extends State<MySignInPage> {
+  final GlobalKey<FormState> _formKey = GlobalKey<FormState>();
+
+  bool eye = true;
+
+  // final alphanumeric = RegExp('r^(?=.*[A-Za-z])(?=.*\d)(?=.*[@$!%*#?&])[A-Za-z\d@$!%*#?&]{8,}$');
+  void vaw() {
+    setState(() {
+      eye = !eye;
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Stack(
+        alignment: Alignment.center,
+        children: [
+          Container(
+            height: MediaQuery.of(context).size.height * 1,
+            width: double.infinity,
+            decoration: const BoxDecoration(
+              image: DecorationImage(
+                image: AssetImage("asset/image/background_image.jpeg"),
+                fit: BoxFit.cover,
+              ),
+            ),
+            child: null,
+            // alignment: Alignment.center,
+          ),
+          ClipRRect(
+            borderRadius: BorderRadius.circular(20),
+            child: BackdropFilter(
+              filter: ImageFilter.blur(sigmaX: 100.0, sigmaY: 100.0),
+              child: SizedBox(
+                height: MediaQuery.of(context).size.height * .7,
+                width: MediaQuery.of(context).size.width * .9,
+                child: Form(
+                  key: _formKey,
+                  child: SingleChildScrollView(
+                    child: Column(
+                      mainAxisAlignment: MainAxisAlignment.center,
+                      children: [
+                        const SizedBox(
+                          height: 65,
+                        ),
+                        const Text(
+                          "SIGN IN",
+                          style: TextStyle(
+                              color: Colors.white,
+                              fontSize: 30,
+                              fontFamily: "san ferif",
+                              fontWeight: FontWeight.bold),
+                        ),
+                        const SizedBox(
+                          height: 70,
+                        ),
+                        SizedBox(
+                          width: MediaQuery.of(context).size.width * .8,
+                          child: TextFormField(
+                            maxLength: 15,
+                            validator: (name) {
+                              if (name == null || name.isEmpty) {
+                                return "Please enter your Name.";
+                              }
+                              return null;
+                            },
+                            style: TextStyle(color: Colors.white),
+                            cursorColor: Colors.white,
+                            decoration: InputDecoration(
+                                counter: Offstage(),
+                                fillColor: Color.fromRGBO(0, 0, 0, 0.3),
```

Comment:
Exact as a static string, color should not be directly used in the class file. Create a separate class and declare color there.